egs2/clarity/enh_2021/local/data.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+set -o pipefail
+
+
+python local/prep_data.py --clarity_root /raid/users/popcornell/Clarity/target_dir/clarity_CEC1_data/clarity_data/
```

Comment:
Could you replace this hard-coded data path with a variable?

Reply:
It would be great if you could add some information about where to download the data.

Reply:
done ! Thanks